apps/admin_panel/assets/src/omg-page-api/index.js

```diff
@@ -393,16 +423,52 @@ class ApiKeyPage extends Component {
   }
 
   render () {
+    const activeTab = this.props.match.params.keyType === 'client' ? 'client' : 'admin'
     return (
       <ApiKeyContainer>
         <TopNavigation
-          title={'Manage API Keys'}
+          divider={this.props.divider}
+          title={'Keys'}
+          description={
+            'These are all keys using in this application. Click each one to view their details.'
```

Comment:
These are the keys that can be used by developers to interact with the eWallet API.